infra/terraform/environments/dev/main.tf

```diff
@@ -234,4 +234,13 @@ module "service" {
       listener_rule_host_header = "ssweb.*"
     }
   }
+
+  job_definitions = {
+    "processQueue" = {
+      job_name = "processQueue",
+      command  = ["/var/www/html/vendor/bin/laminas --container=/var/www/html/config/container-cli.php", "queue:process-queue"],
+      image    = "054614622558.dkr.ecr.eu-west-1.amazonaws.com/vol-app/cli:latest",
```

Comment:
Can we use the version variable (`api_image_tag`) instead, `latest` isn't ideal for a non-local environment.